src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep.java

```diff
@@ -46,7 +46,7 @@
     /**
      * All the sub-workflows as {@link Closure}s, keyed by their names.
      */
-    /*package*/ final Map<String,Closure> closures;
+    /*package*/ final transient Map<String,Closure> closures;
```

Comment:
Because use of the `transient` keyword should be limited to fields which you specifically wish to exclude from serialization in a class which you do expect to be serialized. Its presence here is misleading and therefore should be removed.